app/services/AdministratorService.java

```diff
@@ -1,15 +1,108 @@
 package services;
 
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
+import org.hibernate.criterion.Order;
+import org.hibernate.criterion.Projections;
+
+import models.helpers.AdministratorStatistics;
+import models.helpers.PopularLocation;
+import models.helpers.forms.ImageUploadForm;
+import models.tables.City;
+import models.tables.Reservation;
+import models.tables.Restaurant;
+import models.tables.User;
+import play.api.mvc.MultipartFormData;
+import play.mvc.Http.RequestBody;
+
 /**
  * The type Administrator service.
  */
 @Singleton
 public class AdministratorService extends BaseService {
 
+    
 	@Inject
 	private AdministratorService() { }
+	
+	public String Test(final RequestBody body ) throws Exception {
```

Comment:
Does this have any purpose?

Reply:
Ups, ovo je slucajno zalutalo, iz prethodnog zadatka.